.github/workflows/mariner2.yml

```diff
@@ -0,0 +1,46 @@
+name: build for mariner2
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+
+jobs:
+  build-mariner2:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l -o pipefail {0}
+    container:
+      image: mcr.microsoft.com/cbl-mariner/base/core:2.0
+    steps:
+      - name: install pre-checkout dependencies
+        run: tdnf install --noplugins --skipsignature -y ca-certificates git
+      - name: checkout
+        uses: actions/checkout@v4
+        with:
+          submodules: 'recursive'
+          set-safe-directory: true
+      - name: install dependencies
+        run: |
+          bash .install/mariner2.sh
+      - name: get redis latest
```

Comment:
Check out my comment on the `get_redis.sh` file. we can also get the actual latest release of redis automatically by using [something like this](https://github.com/RediSearch/RediSearch/pull/3904/files#diff-0111532fc2da408313d7e461535ca1e9951b43d57c381c28c9594e58ebf5e3e7)

Reply:
Interesting... it looks much cleaner, but the latest release can also be a patch for older versions (such as `6.2.10`, while the actual latest is `7.2.1` which was released before) - consider this for your PR as well

Reply:
Good catch! That stinks... I'll see if there any other way to get the latest **version**